File: app/src/main/java/com/arny/myapidemo/weather/models/Clouds.java

package com.arny.myapidemo.weather.models;

import com.arny.arnylib.utils.Utility;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Clouds {

    @SerializedName("all")
    @Expose
    private int all;

	@Override
	public String toString() {
		return Utility.getFields(this);
	}

	public int getAll() {
        return all;
    }

    public void setAll(int all) {
        this.all = all;
    }

}

File: app/src/main/java/com/arny/myapidemo/weather/models/Clouds.kt
package com.arny.myapidemo.weather.models

import com.arny.arnylib.utils.Utility
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Clouds {

    @SerializedName("all")
    @Expose
    var all: Int = 0

    override fun toString(): String {
        return Utility.getFields(this)
    }

}
